stable/redis-ha/templates/redis-sentinel-deployment.yaml

```diff
@@ -3,23 +3,14 @@ kind: Deployment
 metadata:
   name: {{ template "fullname" . }}-sentinel
 spec:
-  replicas: {{ .Values.replicas.sentinel }}
+  replicas: {{ .Values.replicas.sentinels }}
   template:
     metadata:
       labels:
-        name: redis-sentinel
-        redis-sentinel: "true"
-        role: sentinel
+        app: "redis-ha"
+        name: "redis-sentinel"
 {{ include "labels.standard" . | indent 8 }}
     spec:
-      {{- if .Values.nodeSelector }}
```

Comment:
You're removing an entire feature of the chart here by removing the nodeSelector parts. Why do that? Some people will like the flexibility of being able to select the nodes on which the Pods should run.

Reply:
Looks like this was added after the fork. I reintroduced the changes.